recipes/qxmpp/all/conanfile.py

```diff
@@ -1,20 +1,27 @@
-from conan import ConanFile
-from conan.tools import files
-from conans import CMake
-import functools
 import os
 
-required_conan_version = ">=1.43.0"
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir, rename, mkdir
+from conan.tools.microsoft import is_msvc
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.54.0"
```

Comment:
```suggestion
required_conan_version = ">=1.56.0 <2 || >= 2.0.6"
```
For the host_versiom syntax